src/test/incremental/hashes/function_interfaces.rs

```diff
@@ -220,7 +220,7 @@ pub fn inline() {}
 pub fn inline_never() {}
 
 #[cfg(not(cfail1))]
-#[rustc_clean(cfg = "cfail2", except = "Hir, HirBody")]
+#[rustc_clean(cfg = "cfail2", except = "Hir, HirBody,promoted_mir")]
```

Comment:
that's odd, there's no promoted, but inlining something makes its promoted mir change?